tox.ini

```diff
@@ -23,20 +23,40 @@ commands = flake8 --max-complexity=10 google test --ignore=E501 --exclude=src-ge
 [testenv:pylint-errors]
 deps = pylint
        -r{toxinidir}/test-requirements.txt
-commands = pylint -f colorized -E google test --ignore=src-gen
+commands = pylint \
+  -f colorized \
+  -E \
+  google test \
+  --ignore=src-gen,_grpc_google_auth.py,test__grpc_google_auth.py
```

Comment:
I thought we only needed this change in `pylint-no-test-deps`; is there a reason to ignore it in the environments that do have a dependency on `test-requirements.txt`?